src/main/java/com/zendesk/maxwell/MaxwellAbstractRowsEvent.java

```diff
@@ -27,6 +30,7 @@
 // and ultimately outputs arrays of json objects representing each row.
 
 public abstract class MaxwellAbstractRowsEvent extends AbstractRowEvent {
+	static final Logger LOGGER = LoggerFactory.getLogger(MaxwellAbstractRowsEvent.class);
```

Comment:
Indeed, they do do that.
I find it odd.. I've only ever seen that for constants in Java of the `public static final int READ_ONLY=0` variety, but good enough for me